conda/meta.yaml

```diff
@@ -11,24 +12,24 @@ source:
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   noarch: python
-  entry_points:
-    - zppy = zppy.__main__:main
 
 requirements:
   host:
-    - python >=3.9
+    - python {{ python_min }}
     - pip
+    - setuptools
```

Comment:
`zppy` gets installed with `setuptools` so this is a necessary dependency.  It used to get bundled with `pip` but that is no longer the case.